tumblr-photo-video-ripper.py

```diff
@@ -201,6 +204,13 @@ def _download_media(self, site, medium_type, start):
                 data = xmltodict.parse(xml_cleaned)
                 posts = data["tumblr"]["posts"]["post"]
                 for post in posts:
+                    if '@reblogged-from-url' in post:
+                        target_folder=target_folder_other
+                        if len (sys.argv) > 2 and sys.argv[2] == '-norepost':
```

Comment:
Better add some help message on this new flag in the main function.

Update README as well please.